cfme/tests/infrastructure/test_instance_analysis.py

```diff
@@ -24,7 +24,7 @@
 
 pytestmark = [pytest.mark.meta(blockers=["GH#ManageIQ/manageiq:6939"],
                                unblock=lambda provider: provider.type != 'rhevm'),
-              pytest.mark.tier(3), test_requirements.smartstate]
+              pytest.mark.tier(3), test_requirements.smartstate, pytest.mark.meta(blockers=1378447)]
```

Comment:
`blockers` must a list. It unfortunately accepts a single element as well right now but it shouldn't.